src/pncounter.rs

```diff
@@ -79,6 +80,14 @@ impl<A: Actor> Causal<A> for PNCounter<A> {
         self.p.forget(&clock);
         self.n.forget(&clock);
     }
+
+    fn read_ctx(&self) -> ReadCtx<(), A> {
+        ReadCtx {
+            add_clock: self.p.inner.clone(),
+            rm_clock: self.n.inner.clone(),
```

Comment:
This doesn't quite work, decrementing and incrementing the counter both "add" information to the PNCounter, I think this is similar to the GCounter where it shouldn't have a `read_ctx`